src/commands/synthetics/utils/internal.ts

```diff
@@ -368,7 +368,6 @@ export const getBasePayload = (test: Test, testOverrides?: UserConfigOverride):
       'retry',
       'startUrlSubstitutionRegex',
       'testTimeout',
-      'tunnel',
```

Comment:
`tunnel` isn't part of the input of this function (`UserConfigOverride`) – it's set [here](https://github.com/DataDog/datadog-ci/blob/c9c4e93346652f426c91a2c41364679698bc492f/src/commands/synthetics/run-tests-lib.ts#L144-L147) on `ServerConfigOverride` directly.